src/ProjectManager.cpp

```diff
@@ -237,15 +232,9 @@ void InitProjectWindow( ProjectWindow &window )
    {
       auto ubs = std::make_unique<wxBoxSizer>(wxVERTICAL);
       ubs->Add( ToolManager::Get( project ).GetTopDock(), 0, wxEXPAND | wxALIGN_TOP );
-      ubs->Add(&ruler, 0, wxEXPAND);
       topPanel->SetSizer(ubs.release());
    }
```

Comment:
Now that this vertical sizer only contains the top tool dock panel, there is no need for it.  The tool panel top dock might be parented directly to `topPanel`, simplifying.

Reply:
I'm not sure what subtle changes in keystroke navigation that might bring.